app/packages/looker-3d/src/hooks/use-pcd-material-controls.tsx

```
@@ -0,0 +1,456 @@
+  const thresholdsLut = useMemo(() => {
+    const allAttributes = geometry.attributes;
+    const attributeNames = Object.keys(allAttributes);
+    return attributeNames
+      .map((attributeName) => {
+        const [min, max] = getMinMaxForAttribute(geometry, attributeName);
+        return {
+          [attributeName]: { min, max },
+        };
+      })
+      .reduce((acc, curr) => {
+        return { ...acc, ...curr };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)